data/data.js

```diff
@@ -0,0 +1,42 @@
+export const playlist = [
+    {
```

Comment:
The different spacing used, should be indented with 2 space tabs.

Reply:
resolved